spec/metaractor_spec.rb

```diff
@@ -236,6 +263,64 @@ def call
       end
     end
 
+    context 'parameter options' do
+      let(:options_class) do
+        Class.new do
+          include Metaractor
+
+          required or: [:token, or: [:recipient_id, :recipient] ]
+          parameter :token, allow_blank: true
+          parameter :thing, required: true
+          required :foo, allow_blank: true
+          optional :bar, default: 'a string'
```

Comment:
What happens if I say `optional :bar, default: "boom", allow_blank: true`? I believe the code does the right thing already by not filtering out `bar`, assigning it with `nil`, and not assigning it with a default value because the key exists for parameters.

Just wanted to confirm that's your thinking as well. `bar == nil` in this case.